### DISCUSSION TITLE
Does security onion work with PPPoE WAN connections?

### DISCUSSION THREAD

**Discussion by cwharvey** [Security-Onion-Solutions/securityonion#7611] [Unsupported Versions] [answered]
The title pretty much sums up the question.

I have a pfSense firewall that is a PPPoE client to my ISP. If I added a security onion appliance between the modem and pfSense WAN port wouldn't security onion only see (presumably encrypted?) PPPoE packets? Is there some way I can make security onion work with my setup?

Thanks

**Comment by dougburks** [comment, marked as answer]
We generally recommend against collecting traffic from the WAN port as this is most likely outside the NAT boundary. From https://docs.securityonion.net/en/2.3/hardware.html#packets:
> If collecting traffic near a NAT boundary, make sure you collect from inside the NAT boundary so that you see the true internal IP addresses.
